tests/test_flatAndIlluminationCorrection.py

```diff
@@ -98,6 +98,30 @@ def testIllum2(self):
     def testIllum3(self):
         self.doIllum(scaling=3.7)
 
+    def testGainAndReadnoise(self):
+        import lsst.afw.image as afwImage
+        from lsst.afw.cameraGeom.testUtils import DetectorWrapper
+        from lsst.ip.isr import IsrTask
+
+        isrTask = IsrTask()
+
+        detector = DetectorWrapper().detector
+        raw = afwImage.ExposureF(detector.getBBox())
+
+        level = 10
+        readNoise = 1
```

Comment:
I was just made aware that `readNoise=1` is not helpful, given the definition of variance below...